main.tf

```diff
@@ -238,7 +238,7 @@ module "kms_key" {
   description             = "KMS key for encrypting Session Logs in S3 and CloudWatch."
   deletion_window_in_days = 10
   enable_key_rotation     = true
-  alias                   = "alias/session_logging_key"
+  alias                   = "alias/session_logging_key_${var.stage}"
```

Comment:
Similar to the below, if you want to create a KMS key for each stage/environment that you're using this module with then I suggest creating it externally and passing into the module via the `session_logging_kms_key_arn` variable.

Reply:
@nordinh mind checking out my comment here and reverting this change? Creating a single KMS Key externally is the way to go in the case that you're utilizing this module for multiple environments in a single account.